### PR TITLE
[Listener] processing StandingSellOrderBatch events

### PR DESCRIPTION
**Pull request by fleupold**
This PR contains logic for processing *StandingSellOrderBatch* events. It depends on https://github.com/gnosis/dex-contracts/pull/97 and assumes that the order details are emitted as blobs.

For now it relies on unit tests to ensure functionality is correct.

## Test Plan
Once depending PR is landed, update submodule and run placeStandingOrder script. Observe that standing orders are added to the database.

### CODE REVIEW

File: test/e2e-tests-standing-order.sh [hunk 1]
```diff
@@ -0,0 +1,10 @@
+#!/bin/bash
+set -e
+
+cd dex-contracts/
+
+# Place standing order in current Auction (accountId, buyToken, sellToken, minBuy, maxSell)
+truffle exec scripts/standing_order.js 0 1 2 12 12
```
Comment (bh2smith) [inline test/e2e-tests-standing-order.sh]: I didn't realize we had these scripts already! or is this a preemptive change?

Comment (fleupold) [reply test/e2e-tests-standing-order.sh]: https://github.com/gnosis/dex-contracts/pull/133

**Comment by josojo** [comment]
Nice, the PR is already looking good. I did the test plan and it all worked smoothly.

**Comment by fleupold** [comment]
Cool, I think it's ready for review

**Comment by bh2smith** [comment]
What about the conflicts @fleupold ? Are they a concern before merge?

File: event_listener/dfusion_db/models.py [hunk 2]
```diff
@@ -180,3 +180,68 @@ def from_dictionary(cls, data: Dict[str, Any], num_tokens: int, num_orders: int)
...
+            int(data['validFromAuctionId']),
+            list(map(OrderDetails.from_bytes, packed_order_array))
+        )
+
+    def to_dictionary(self) -> Dict[str, Any]:
+        return {
+            'accountId': self.account_id,
+            'batchIndex': self.batch_index,
+            'validFromAuctionId': self.valid_from_auction_id,
+            'orders': list(map(lambda o: o.to_dictionary(), self.orders))
```
Comment (bh2smith) [inline event_listener/dfusion_db/models.py]:
```suggestion
            'orders': list(map(lambda order: order.to_dictionary(), self.orders))
```

File: event_listener/dfusion_db/models.py [hunk 3]
```diff
@@ -180,3 +180,68 @@ def from_dictionary(cls, data: Dict[str, Any], num_tokens: int, num_orders: int)
...
+    valid_from_auction_id: int
+    orders: List[OrderDetails]
+
+    @classmethod
+    def from_dictionary(cls, data: Dict[str, Any]) -> "StandingOrder":
+        event_fields = ('currentBatchIndex', 'accountId', 'packedOrders', 'validFromAuctionId')
+        assert all(k in data for k in event_fields), "Unexpected Event Keys: got {}".format(data.keys())
+
+        packed_order_str = data['packedOrders']
+        packed_order_array = [packed_order_str[i: i + 52] for i in range(0, len(packed_order_str), 52)]
```
Comment (bh2smith) [inline event_listener/dfusion_db/models.py]: Safe to assume the significance of 52 here is that 52 = 26 * 2.

**Pull request merged**